spectec/spec/7-module.watsup

```diff
@@ -0,0 +1,236 @@
+;; Allocation
+
+def $alloc_import(moduleinst, import*, externval*) : moduleinst
```

Comment:
I suppose we shouldn't call this allocation, since it doesn't create a new instance or extend the store. 

To be honest, I was hoping we can avoid such verbose manual iteration and do it with *-notation like in the spec.